### ISSUE TITLE
Test Accessibility: Testimonials Add/ Edit

### ISSUE THREAD

**Issue by naragurgel** [naragurgel/serenity_oils_aroma#55]
# Steps to take

1. Open a new incognito tab in the web browser.
2. Enter the URL of the [testimonials add](https://serenity-oils-3beb26e98281.herokuapp.com/testimonials/add/)
3. Right-click anywhere on the page and select "Inspect" to open the browser's developer tools.
4. In the developer tools panel, navigate to the "Audits" tab.
5. Click on the "Run audits" button to initiate the Lighthouse audit.

# Expected Outcome

- [x]  Accessibility should be 90+

**Comment by naragurgel** [comment]
![image](https://github.com/naragurgel/serenity_oils_aroma/assets/112726044/c8c06806-7eae-4fd4-a540-7ff4c53a64d2)

The add and edit testimonial pages have slightly lower accessibility scores due to headers out of order and iframes missing titles. These errors are beyond my control as the rendering of summernote rich text fields is not an interface I wrote.
